web/src/composebox_typeahead.ts

```diff
@@ -900,32 +887,6 @@ export function get_candidates(
         return typeahead_helper.sort_slash_commands(matches_list, token);
     }
 
-    if (ALLOWED_MARKDOWN_FEATURES.stream && current_token.startsWith("#")) {
```

Comment:
Oh, it would probably be better to have a prep commit swapping these two blocks, to make it clear when code is being changed vs moved.